examples/regression/1-quickstart/plot_heteroscedastic_1d_data.py

```diff
@@ -0,0 +1,197 @@
+"""
```

Comment:
It would be great if we could compare the coverage and the averaged size of prediction intervals between the methods. It would highlight the need for CQR on heteroscedastic data. To do this, you would have to split the training set created by your custom function into training and test sizes.

Reply:
I think it's best for now to keep this table in the tutorial and just do plotting. Maybe add this as an issue for further implementation?

Reply:
Agreed ! I added it in our backlog.